fluka/tests/fludag_unit_tests.cpp

```diff
@@ -45,7 +44,7 @@ class FluDAGTest : public ::testing::Test
     // Default h5m file for testing
     std::string infile = "slabs.h5m";
 
-    rloadval = DAG->load_file(infile.c_str(), 0.0 );
```

Comment:
This was changed in MOAB, since the faceting tolerance argument no longer serves any useful purpose. Originally this passed the ftol onto CGM to get facets.